MdeModulePkg/Universal/Variable/UefiVariablePolicy/ReadMe.md

```diff
@@ -0,0 +1,354 @@
+---
+title:      UEFI Variable Policy Whitepaper
+version:    1.0 RFC
+copyright:  Copyright (c) Microsoft Corporation.
+---
+
+# UEFI Variable Policy
```

Comment:
can you add a section on testing and provide brief description of the two tests.  Host and target.